share/plugins/lilyrics/lilyrics.qml

```diff
@@ -87,7 +87,7 @@ MuseScore {
             anchors.bottomMargin: 5
             anchors.rightMargin: 10
             height: 50
-            tooltip: qsTr("Copy the text to the score");
+            //tooltip: qsTr("Copy the text to the score");
```

Comment:
Perhaps you can use `toolTipTitle` property here

Reply:
fixed